src/Nelmio/Alice/Fixtures/Parser/Methods/Base.php

```diff
@@ -102,17 +102,18 @@ protected function processIncludes($data, $filename)
     /**
      * @param array $data
      * @param array $includeData
+     * @return array
      */
     protected function mergeIncludeData($data, $includeData)
     {
-        foreach ($includeData as $class => $fixtures) {
-            if (isset($data[$class])) {
-                $data[$class] = array_merge($fixtures, $data[$class]);
+        foreach ($data as $class => $fixtures) {
+            if (isset($includeData[$class])) {
+                $includeData[$class] = array_merge($includeData[$class], $fixtures);
             } else {
-                $data[$class] = $fixtures;
+                $includeData[$class] = $fixtures;
             }
         }
 
-        return $data;
+        return $includeData;
```

Comment:
@piribes I feel like changing `$includeData` makes the intention here less clear, since conceptually `$includeData` should be appended to `$data`. Instead, let's just create a new variable, `$newData` or something, and return it. Does that make sense?